src/Products/Common/Resources/Resources.cs

```diff
@@ -47,7 +46,7 @@ public static string GetFreeFileName(string directory, string fileName)
         /// </summary>
         /// <param name="ex">Exception</param>
         /// <returns>ExceptionEntity</returns>
-        public ExceptionEntity GenerateException(System.Exception ex)
+        public ExceptionEntity GenerateException(Exception ex)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'GenerateException' a static method.](https://app.codacy.com/gh/groupdocs-editor/GroupDocs.Editor-for-.NET-WebForms/pullRequest?prid=5153088)